### ISSUE TITLE
please split qa instance rename from qa cluster rename

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#397]
Originally reported of Google Code with ID 341.

```
If instance and cluster live on different netblocks the same rename name can't be used. There must be the possibility of using different ones.
```
Originally added on 2013-01-09 08:39:39 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed on stable-2.6:

commit 69bc7a3
Author: Michael Hanselmann <hansmi@google.com>
Date:   Fri Jan 11 14:59:03 2013 +0100

    QA: Use instance name for rename test
    
    Reported in issue 341. In some setups the instance live in a different
    netblock from the cluster. Therefore a the configuration-global “rename”
    name shouldn't be used for them. Instead another instance name is used.
    
    Signed-off-by: Michael Hanselmann <hansmi@google.com>
    Reviewed-by: Guido Trotter <ultrotter@google.com>


```
Originally added on 2013-01-11 16:20:48 +0000 UTC.

Mentioned Issue with GoogleCode ID 341 has been migrated to GitHub with ID #397.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
